Projects/l0/Sniffer/inc/gpio.h

```diff
@@ -21,25 +21,26 @@
 #ifndef __gpio_H
 #define __gpio_H
 #ifdef __cplusplus
- extern "C" {
+extern "C"
+{

```

Comment:
you shouldn't have to lint those files because they are generated by cube.
As you can see in Examples/.clang-format-ignore, those files should be ignored by the linter.
If you need to check the linting the same way Github is doing you can run : 
python Examples/.github/workflows/run-clang-format.py

Reply:
You are right ,I did the linting automatically to the files after I developped them and I didnt notice that it proposed changes to this file too